src/webapp/js/components/locations.js

```diff
@@ -126,17 +126,18 @@ class Locations extends Component {
     this.handleTabSelect = this.handleTabSelect.bind(this);
   }
 
+  componentWillMount() {
+    this.setState({ selectedIndex: Number.parseInt(this.props.match.params.tabIndex, 10) });
+    if (!this.props.match.params.tabIndex) {
```

Comment:
you should change it to if (this.props.match.params.tabIndex) and move this.setState({ selectedIndex: Number.parseInt(this.props.match.params.tabIndex, 10) }); inside this if

Reply:
changed